modules/http/src/test/scala/higherkindness/mu/rpc/http/GreeterServices.scala

```diff
@@ -17,6 +17,7 @@
 package higherkindness.mu.rpc.http
 
 import higherkindness.mu.rpc.protocol._
+import higherkindness.mu.http._
```

Comment:
Maybe for consistency, we want to have an additional subpackage `protocol`, like in the `rpc` case.

Reply:
Done at https://github.com/higherkindness/mu/pull/581/commits/b8be4b74924844d2ecddcb626dd2477db816207c